lib/models/Post.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require('mongoose');
+
+const schema = new mongoose.Schema({
+  user: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'User',
+    required: true
+  },
+  photoURL: {
+    type: String,
+    required: true
+  },
+  caption: {
+    type: String,
+    required: true
+  },
+  tags: [String]
```

Comment:
Another way you could have rewrote line 17 to allow for more room for things like require:

tags: [{
type: String,
required: true
}]